src/main/java/com/sda/zdjavapol107/checkers/CheckersLoad.java

```diff
@@ -0,0 +1,62 @@
+package com.sda.zdjavapol107.checkers;
+
+import com.sda.zdjavapol107.HibernateFactory;
+import com.sda.zdjavapol107.checkers.dao.*;
+import com.sda.zdjavapol107.checkers.model.*;
+import org.hibernate.SessionFactory;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CheckersLoad {
+
+    SessionFactory sessionFactory;
+    PlayerDao playerDao;
+    GameDao gameDao;
+    GamePeacesDao gamePeacesDao;
+    MoveDao moveDao;
+    TournamentDao tournamentDao;
+
+    CheckersLoad(SessionFactory sessionFactory) {
+        this.sessionFactory = sessionFactory;
+        playerDao = new PlayerDao(sessionFactory);
+        gameDao = new GameDao(sessionFactory);
+        gamePeacesDao  = new GamePeacesDao(sessionFactory);
+        moveDao  = new MoveDao(sessionFactory);
+        tournamentDao = new TournamentDao(sessionFactory);
+    }
+
+    public CheckersBoard loadLastGame() {
+        Tournament tournament = tournamentDao.getLastTournament();
+        List<Game> gameList = gameDao.getGamesByTournamentId(tournament.getId());
+        Game game = loadLatestGame(gameList);
+//        List<Player> playerList = playerDao.getPlayersByGameId(tournament.getId());
+        List<GamePeaces> gamePeacesList = gamePeacesDao.getGamePeacesByGameId(game.getId());
+        List<Move> moveList = moveDao.getMovesByGameId(game.getId());
+
+        CheckersTournament checkersTournament = new CheckersTournament(tournament.getPlayerOneScore(), tournament.getPlayerTwoScore());
+//        List<CheckersPlayer> checkersPlayerList = convertPlayersToChPlayers(playerList);
+
+        return null;
+    }
+
+    private Game loadLatestGame(List<Game> gameList) {
```

Comment:
to jest fajne miejsce na uzycie Optionala :) 
poza tym fajnie mozna to zrobic streamem, mozesz sprobowac (chociaz to raczej w ramach treningu uzycie streama nic nie zmieni :) )